website/docs/docs/deploy/hybrid-setup.md

```diff
@@ -157,5 +157,5 @@ Now that you've integrated <Constant name="core" /> artifacts with your <Constan
 
 
 - Collaborate with <Constant name="cloud" /> users by enabling them to visualize and perform [cross-project references](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref) to dbt models that live in Core projects.
-- (Coming soon) New users interested in the [<Constant name="visual_editor" />](/docs/cloud/visual-editor) can build off of dbt models already created by a central data team in <Constant name="core" /> rather than having to start from scratch.
+- (Coming soon) New users interested in the [<Constant name="visual_editor" />](//docs/cloud/canvas) can build off of dbt models already created by a central data team in <Constant name="core" /> rather than having to start from scratch.
```

Comment:
```suggestion
- (Coming soon) New users interested in the [<Constant name="visual_editor" />](/docs/cloud/canvas) can build off of dbt models already created by a central data team in <Constant name="core" /> rather than having to start from scratch.
```